paperqa/config.py

```diff
@@ -432,6 +449,43 @@ def from_name(
 
         raise FileNotFoundError(f"No configuration file found for {config_name}")
 
+    def get_llm(self) -> LiteLLMModel:
+        return LiteLLMModel(
+            name=self.llm,
+            config=self.llm_config
+            or {
+                "model_list": [
```

Comment:
Can you add a brief inline comment what this structure is supposed to match?

I also see it's repeated, should we make a private helper function for this?

Reply:
yea good call -- added